src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/CreateSDORespondent1NotificationHandler.java

```diff
@@ -55,34 +58,23 @@ public List<CaseEvent> handledEvents() {
     private CallbackResponse notifyRespondentSolicitor1SDOTriggered(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
 
-        notificationService.sendMail(
-            caseData.getRespondentSolicitor1EmailAddress(),
-            caseData.getCaseAccessCategory() == CaseCategory.SPEC_CLAIM
-                ? notificationsProperties.getSdoOrderedSpec()
-                : notificationsProperties.getSdoOrdered(),
-            addProperties(caseData),
-            String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference())
-        );
+        if (NOTIFY_RESPONDENT_SOLICITOR1_SDO_TRIGGERED.name().equals(callbackParams.getRequest().getEventId())) {
+            if (caseData.getSpecRespondent1Represented() == YesOrNo.YES
+                || caseData.getRespondent1Represented() == YesOrNo.YES) {
```

Comment:
not that I know of. Afaik, CaseData is barely anything more than a POJO

Reply:
it does have methods there that are reusable and are to do with the case entity if you have a look

Reply:
for example i found this in CaseData that you could use instead: 
` public YesOrNo getRespondent1Represented() {
        return Stream.of(
                respondent1Represented,
                specRespondent1Represented
            )
            .filter(Objects::nonNull)
            .findFirst().orElse(null);
    }

    public YesOrNo getRespondent2Represented() {
        return Stream.of(
                respondent2Represented,
                specRespondent2Represented
            )
            .filter(Objects::nonNull)
            .findFirst().orElse(null);
    }
`

Reply:
or you could create your own boolean  method that would use those ones so that your handler does not have to know about the YesNo enum

Reply:
for example you could add this to CaseData: 
 ```
 @JsonIgnore
    public boolean isRespondent1Represented(){
        return getRespondent1Represented() == YesOrNo.YES;
    }

```
And then use it in your handler. I think it would be cleaner and also would avoid code repetitions when we need to find out the same flag in other places